fips202/fips202x.h

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: Apache-2.0
+#ifndef FIPS_202X_H
+#define FIPS_202X_H
+
+#ifndef KECCAK_WAY
+#define KECCAK_WAY 7
+#endif
+
+#include <stdint.h>
+#include <stddef.h>
+
+typedef struct
```

Comment:
Why do we need this?

Reply:
the compiler warning that it doesn't understand `uint64_t` (added `stdint`) and `size_t` (added `stddef`).